client/client.go

```diff
@@ -19,24 +19,37 @@ type Client interface {
 
 	// TrackOperation tracks custom operation
 	TrackOperation(section string, operation bucket.MetricOperation, t timer.Timer, success bool) Client
+
 	// TrackOperationN tracks custom operation with n diff
 	TrackOperationN(section string, operation bucket.MetricOperation, t timer.Timer, n int, success bool) Client
 
 	// TrackMetric tracks custom metric, w/out ok/fail additional sections
 	TrackMetric(section string, operation bucket.MetricOperation) Client
+
+	// TrackMetricWithLabels tracks custom metric with added labels, w/out ok/fail additional sections
+	TrackMetricWithLabels(section string, operation bucket.MetricOperation, labels map[string]string) Client
```

Comment:
too many implementation-specific methods in the interface, maybe we need to rethink this a bit, maybe even by breaking interface, e.g. replace `bucket.MetricOperation` with the builder like we do for timer, need to think more about it